### PR TITLE
Add initial code

### PR DESCRIPTION
**Pull request by palcu**
# Need

As devs
We need to start from somewhere
# Deliverables
- [x] update README.md with:
  - [x] project description
  - [x] technologies used
  - [x] goals
  - [x] folder layout
- [x] research boilerplate for a React project

We will use [essential-react](https://github.com/pheuter/essential-react).
- [x] remove useless stuff from there
- [x] configure jshint
- [x] configure LESS
- [x] react-bootstrap
- [ ] write those fucking tests

### CODE REVIEW

**Comment by palcu** [comment]
@hurrycane please code review

File: Gulpfile.js [hunk 1]
```diff
@@ -0,0 +1,13 @@
+var gulp = require('gulp'),
+    eslint = require('gulp-eslint');
+
+gulp.task('lint', function () {
+    return gulp.src('src/**/*.{js,jsx}')
```
Comment (hurrycane) [inline Gulpfile.js]: be consistent in style. Either 2 or 4 spaces. Choose one and stick to it.

File: README.md [hunk 2]
```diff
@@ -1,2 +1,32 @@
 # infoeducatie-react
-React frontend for infoeducatie-api
+
+React frontend for our [backend API](https://github.com/infoeducatie/infoeducatie-api).
```
Comment (hurrycane) [inline README.md:3]: Any word about dependencies here? For karma we need phantom for example.

**Comment by hurrycane** [comment]
I am not that proficient in react to do a proper code review but I can see some of code style inconsistency. Try doing a self review for those before you merge.

For my unexperienced react eyes it looks good.

**Pull request merged**